analyze_openapi_changes.py

```
@@ -0,0 +1,180 @@
+
+def save_markdown(summary_md: str, output_dir: str, repo_name: str, date: str) -> None:
+    os.makedirs(output_dir, exist_ok=True)
+    
+    md_file = os.path.join(output_dir, f"{repo_name}-{date}-open-api-diff.md")
+
+    with open(md_file, 'w') as f:
+        f.write(summary_md)
+
+    print(f"Markdown summary saved to {md_file}")
+
+def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)